e2e/tests/variables.bash

```diff
@@ -2,9 +2,9 @@
 
 ### COMMON VARIABLES ###
 : ${DOCKER_REGISTRY:="dispatchframework"}
-: ${BASE_IMAGE_PYTHON3:="python3-base:0.0.9"}
-: ${BASE_IMAGE_NODEJS6:="nodejs-base:0.0.9"}
-: ${BASE_IMAGE_POWERSHELL:="powershell-base:0.0.10"}
-: ${BASE_IMAGE_JAVA:="java-base:0.0.10"}
+: ${BASE_IMAGE_PYTHON3:="python3-base:0.0.13"}
```

Comment:
Should it change to knative version?

Reply:
We will have to do that with your change. The e2e tests will fail if we don't have your changes as well.